packages/react-scripts/package.json

```diff
@@ -79,7 +78,6 @@
     "semver": "7.3.2",
     "style-loader": "1.3.0",
     "terser-webpack-plugin": "4.2.3",
-    "ts-pnp": "1.2.0",
```

Comment:
This is used directly in the code and therefor can't be removed https://github.com/facebook/create-react-app/blob/b36d1ea1aa1458907839da9dd5028a372f9d48d0/packages/react-scripts/config/pnpTs.js#L11